src/comments.ts

```diff
@@ -60,7 +60,7 @@ export const SuggestTesting = deletedWhenNotPresent("suggest-testing", tag =>
 export const PingReviewers = (names: readonly string[], reviewLink: string) => ({
     tag: "pinging-reviewers",
     status: txt`
-        |🔔 ${names.map(n => `@${n}`).join(" ")} — please [review this PR](${reviewLink}) in the
+        |🔔 Pinging owners — please [review this PR](${reviewLink}) in the
```

Comment:
I believe this is the initial comment on PRs, which should ping.

Reply:
Yes you're right, this was a mistake. 🙏